content/en/tutorial/01-vdom.md

```diff
@@ -185,26 +202,26 @@ useResult(function(result) {
 
 
 ```jsx:repl-initial
-import { render } from 'preact';
+import { createElement, render } from 'preact';
```

Comment:
May or may not throw users off, but I figure it might help to be a bit consistent w/ JSX and importing `h`/`createElement`. Our other snippets on this page do import it.